Performance-Tracing.md

```diff
@@ -0,0 +1,156 @@
+# Performance Tracing
+
+TypeScript 4.1 introduced a `generateTrace` flag to make it easier to identify the parts of a program that are taking the most time to compile.
+
+## Warnings
+
+This feature is still experimental and will likely change significantly in TS 4.2.
+In particular, though it would be easy and helpful to post-process the trace files, you should assume that your post-processing tools will be broken by TS 4.2.
+
+Trace files are basically useless without the underlying code, so there's little reason to share one with anyone who can't access your code.
+If you do share them, please consider zipping them - they compress very well.
+
+## Getting Started
+
+At present, the output is not very user friendly, but here are some steps you can take to identify the parts of your code that are slowing the compiler down.
+The goal of this process is to be able to extract a reduced repro for which you can file an [issue](https://github.com/microsoft/TypeScript/issues).
+
+1.  This document is specific to TS 4.1 RC and TS 4.1, so you'll need to install one of those.
+    The easiest way is to `npm install typescript@rc` ( or `typescript@4.1`, once it is released).
+    There's no need to check in a change to your repo - you can just use it locally while you do your tracing.
+
+2.  Rebuild your project with the newly installed version of Typescript.
+    Odds are you're going to get some new compiler errors.
+    Ideally, you want to fix these new errors correctly, so that the code you trace will be realistic.
+    If that is not feasible, prefer `@ts-ignore` to `any`, because it will cause less short-circuiting.
+
+3.  Once your project builds without errors, compile your project with `tsc` and `-generateTrace some_directory`.
```

Comment:
`-generateTrace` -> `--generateTrace` (I see a single dash)

Also in `-incremental` below

Reply:
Either works technically, but I prefer 2 dashes just to give people a clue.

Reply:
We accept both and I believe the help text suggests just one.

Reply:
The help text uses two, so I'll match that.

Reply:
I didn't even know that it accepts a single dash.  Is this some node convention?  (Usually `-ab` is equivalent to `-a -b`, and therefore long options require `--`)